python/lsst/ts/eas/m1m3ts_model.py

```diff
@@ -359,9 +491,16 @@ async def wait_for_noon(
                 ):
                     self.log.info(
                         "Noon M1M3TS is set based on twilight temperature: "
-                        f"{self.weather_model.last_twilight_temperature}"
+                        f"{self.weather_model.last_twilight_temperature:.2f} "
+                        f"(with floor at {self.setpoint_lower_limit:.2f})"
                     )
                     await self.apply_setpoints(
                         m1m3ts_remote=m1m3ts_remote,
                         setpoint=self.weather_model.last_twilight_temperature,
+                        enforce_limit=False,
```

Comment:
Shouldn't that be set to True? Even for the noon temperature setpoint, we would like to enforce the limit, right?